packages/studio/src/components/ErrorComponentPreview.tsx

```diff
@@ -1,14 +1,13 @@
-import { ErrorComponentState } from "@yext/studio-plugin";
 import { useCallback, useMemo, useRef, useState } from "react";
 import { v4 } from "uuid";
 import ErrorBoundary from "./common/ErrorBoundary";
-import { Tooltip } from "react-tooltip";
 
 export default function ErrorComponentPreview(props: {
-  errorComponentState: ErrorComponentState;
+  errorComponentState: string;
```

Comment:
Could we preserve the input type of the function to stay as type `ErrorComponentState`?  I think we can still pass in the error state and extract the message from the error state within this function!